en/development/routing.rst

```diff
@@ -592,7 +565,7 @@ you to define name prefixes in each scope::
 You can also use the ``_namePrefix`` option inside nested scopes and it works as
 you'd expect::
 
-    Router::plugin('Contacts', ['_namePrefix' => 'contacts:'], function ($routes) {
+    Router:{plugin}('Contacts', ['_namePrefix' => 'contacts:'], function ($routes) {
```

Comment:
I don't think this should have changed.

Reply:
Bad search replaces :slightly_smiling_face: